metricbeat/helper/prometheus/metric.go

```diff
@@ -102,6 +103,13 @@ func OpMultiplyBuckets(multiplier float64) MetricOption {
 	}
 }
 
+// OpSetSuffix extends the field's name with the given suffix
+func OpSetSuffix(suffix string) MetricOption {
```

Comment:
Should we name or document this in a way that indicates that the suffix is only added if the value is a float and not an histogram?